tests/helpers/webbtest.go

```diff
@@ -125,7 +120,7 @@ func BgpDeleteNbr(nbrs []*BgpNeighbor) *oc.NetworkInstance_Protocol_Bgp {
 func VerifyPortsUp(t *testing.T, dev *ondatra.Device) {
 	t.Helper()
 	for _, p := range dev.Ports() {
-		status := dev.Telemetry().Interface(InterfaceMap[p.Name()]).OperStatus().Get(t)
+		status := dev.Telemetry().Interface(p.Name()).OperStatus().Get(t)
```

Comment:
Just an query, whether we can use Telemetry() in our Athena-G, now? From this diff I could not trace out the caller of "VerifyPortsUp"

Reply:
its just, checking DUT interfaces are UP or not, that's it via DUT gNMI